howso/client/exceptions.py

```diff
@@ -91,23 +159,56 @@ def from_dict(cls, obj):
         if obj is None:
             obj = {}
 
-        title = obj.get('title', '')
-        detail = obj.get('detail', '')
+        default_msg = 'An unknown error occurred.'
+        detail = obj.get('detail') or default_msg
```

Comment:
`detail = obj.get('detail', default_msg)`?

Reply:
In the case where `detail` is null, this ensures a message is actually used